custom/icds_reports/ucr/data_sources/person_cases_v3.json

```diff
@@ -615,9 +623,15 @@
           },
           "default": {
             "type": "constant",
-            "constant": null
+            "constant": 0
```

Comment:
is it possible to avoid changing the indicators in this data source? ie, to make expressions that take into account what null means. rebuilding this data source is going to be incredibly painful, and in theory all existing expression already handle this behavior

Reply:
Sunaina/Akshita can you respond to this?